linalg/operator.cpp

```diff
@@ -645,25 +645,106 @@ void ConstrainedOperator::ConstrainedMult(const Vector &x, Vector &y,
    }
 }
 
+void ConstrainedOperator::ConstrainedAbsMult(const Vector &x, Vector &y,
```

Comment:
The implementation should be merged with `ConstrainedOperator::ConstrainedMult` since it basically follows the same pattern